### PR TITLE
feat: obtain repository settings file

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #5 
Created `RepoSettings` to hide all the logic with flat/org layout and yaml/yml extension while searching for repository settings file by its name. Also:
- added runtime `NotFoundException` and corresponding handler for it
- simplified `GetRepository` class to use new `RepoSettings` class
- added method to get attribute name into `RequestAttr.Standard`

### CODE REVIEW

**Review by g4s8 — CHANGES_REQUESTED** [review]
@olenagerasimova just minor comment

File: src/main/java/com/artipie/front/RequestAttr.java [hunk 1]
```diff
@@ -57,6 +57,14 @@
             this.name = name;
         }
 
+        /**
+         * Attribute name.
+         * @return The name
+         */
+        public String attrName() {
```
[the remark this answers is not in the document]
Comment (olenagerasimova) [reply src/main/java/com/artipie/front/RequestAttr.java:64]: Enums already have final method `name()` to return item names, so I cannot rename this method(

**Comment by olenagerasimova** [comment]
@g4s8 answered and fixed checkstyle after merging master, take a look please

**Review by g4s8 — APPROVED** [review]
@olenagerasimova ok, thanks

**Pull request merged**